packages/calcite-components/src/components/input-date-picker/input-date-picker.scss

```diff
@@ -24,9 +24,7 @@
 @include disabled();
 
 .calendar-picker-wrapper {
-  @apply static
-    w-full
-    shadow-none;
+  @apply shadow-none static;
```

Comment:
is static needed? its the default position.

Reply:
Nope. Good catch. 🎣 Will remove!